lib/retriable/exponential_backoff.rb

```diff
@@ -25,19 +25,16 @@ def initialize(opts = {})
 
     def intervals
       intervals = Array.new(tries) do |iteration|
-        [base_interval * multiplier**iteration, max_interval].min
+        [base_interval * multiplier ** iteration, max_interval].min
       end
 
-      return intervals if rand_factor.zero?
```

Comment:
I'm going to keep this line. The reason I like it is because if rand_factor is 0, basically, don't randomize. This line basically does that from a logical step point of view: if it's 0, don't call `randomize`. It skips some math, `map` allocates memory for a new Array. I know it's super minimal, but this logically makes sense to me.